client/galaxy/scripts/layout/page.js

```diff
@@ -16,7 +16,12 @@ define( [ 'layout/masthead', 'layout/panel', 'mvc/ui/ui-modal' ], function( Mast
 
             // attach global objects, build mastheads
             Galaxy.modal = this.modal = new Modal.View();
-            Galaxy.display = this.display = function( view ) { self.center.display( view ) };
+            Galaxy.display = this.display = function( view ) {
+                if ( view.title ){
+                    window.document.title = "Galaxy | " + view.title;
```

Comment:
would this be the place to include the `app.config.brand` if present? (as in https://github.com/galaxyproject/galaxy/pull/4272)

Reply:
Yep, I like that.  I was also talking to @guerler about whether or not we should just set it to "Galaxy | <brand>" if there is no view title set.  I'm looking for cases where we may *not* want to do that, and if I can't find one I'll add that too.